RELEASES.md

```diff
@@ -81,6 +81,54 @@ Beta releases serve as a preview of the upcoming stable release and are meant fo
 ### Release Candidates (rc)
 Release candidates are the final step before the stable release. These versions are considered feature complete, with all proposed features implemented and bugs addressed. The primary purpose of an RC is to ensure that there are no critical issues that were missed during beta testing. If no significant problems are identified in an RC, it can be promoted to a stable release. However, if issues are found, they are addressed, and a new RC is issued for testing.
 
+## Experimental Modules
+When adding new features to Quickstart, particularly large or complicated
+features, we usually add these modules as "experimental" to indicate to site
+owners that these modules are not fully tested or feature complete and will
+likely experience significant or breaking changes. These modules should only be
+used by site owners that are actively tracking Quickstart development and are
+prepared to resolve the consequences of breaking changes. 
```

Comment:
I've been thinking that the process for moving a feature from experimental should include a decision about whether to leave the functionality in a free-standing/optional module or move it into an existing AZQS module.

Drupal core does this.  Core's single directory components feature started out as an optional experimental module but when they moved it to stable it was actually integrated into the core theme system (no longer a separate module and no longer optional).  I think an approach like this would probably make sense for some of our experimental features as well.

I started to try to draft some changes to this documentation along these lines here but didn't get very far yet:

```suggestion
## Experimental Features
When adding new features to Quickstart, particularly large or complicated
features, we usually encapsulate these in modules labeld as "experimental" to indicate to site
owners that these feature is not fully tested or feature-complete and will
likely undergo significant or breaking changes. These features should only be
used by site owners that are actively tracking Quickstart development and are
prepared to resolve the consequences of breaking changes. 
```

Reply:
`feature is` -> `features are`